src/pages/Editor.vue

```diff
@@ -1,104 +1,125 @@
-          <div class="flex relative justify-center">
-            <img src="@/assets/images/plain.svg" />
-            <img src="@/assets/images/play.svg" class="absolute place-self-center" />
+        <div class="justify-center">
+          <!--- video preview -->
+          <div v-if="!isVideoIdValid" class="flex justify-center">
+            <div class="flex relative justify-center">
+              <img src="@/assets/images/plain.svg" />
+              <img src="@/assets/images/play.svg" class="absolute place-self-center" />
+            </div>
           </div>
-        </div>
-        <div v-else>
-          <video-player
-            :videoId="videoId"
-            :plyrConfig="plyrConfig"
-            @update="videoTimestampUpdated"
-            @ready="playerReady"
-            @play="playerPlayed"
-            ref="playerObj"
-          ></video-player>
+          <div v-else>
+            <video-player
+              :videoId="videoId"
+              :plyrConfig="plyrConfig"
+              @update="videoTimestampUpdated"
+              @ready="playerReady"
+              @play="playerPlayed"
+              ref="playerObj"
+            ></video-player>
 
-          <!--- slider with question markers -->
-          <slider-with-markers
-            :end="videoDuration"
-            :step="sliderStep"
-            v-model:value="currentTimestamp"
-            v-model:markerPositions="itemTimestamps"
-            @marker-selected="itemSelected"
-            @marker-drag-end="itemMarkerTimestampDragEnd"
-            @update="sliderUpdated"
-            ref="slider"
-          ></slider-with-markers>
+            <!--- slider with question markers -->
+            <slider-with-markers
+              :end="videoDuration"
+              :step="sliderStep"
+              v-model:value="currentTimestamp"
+              v-model:markerPositions="itemTimestamps"
+              @marker-selected="itemSelected"
+              @marker-drag-end="itemMarkerTimestampDragEnd"
+              @update="sliderUpdated"
+              ref="slider"
+            ></slider-with-markers>
+          </div>
         </div>
-      </div>
 
-      <!--- buttons -->
-      <div class="flex justify-center mt-10">
-        <Button
-          label="Publish Plio"
-          class="p-button-success"
-          :disabled="!isVideoIdValid"
-          v-tooltip.bottom="'Click to publish plio'"
-        />
+        <!--- buttons -->
+        <div class="flex justify-between mt-10">
+          <icon-button
+            :titleConfig="backButtonTitleConfig"
+            :iconConfig="backButtonIconConfig"
+            :buttonClass="backButtonClass"
+            @click="returnToHome"
+          ></icon-button>
+          <icon-button
+            :titleConfig="publishButtonTitleConfig"
+            :class="publishButtonClass"
+            v-tooltip.right="publishButtonTooltip"
+            @click="publishButtonClicked"
+          ></icon-button>
```

Comment:
also mention in one line what these buttons are
like comments "video link", "item editor", "video preview"